xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/editobject.vm

```diff
@@ -36,7 +36,7 @@
     #end
     #foreach ($prop in $properties)
       <dt class="label#if($prop.isDisabled()) disabled#end"><label for="${escapetool.xml($class.name)}_${obj.number}_${escapetool.xml($prop.name)}">${escapetool.xml($prop.translatedPrettyName)}</label></dt>
-      <dd#if ($prop.isDisabled()) class="disabled"#end>$doc.display($prop.name, "edit", $obj)</dd>
+      <dd#if ($prop.isDisabled()) class="disabled"#end>#if ("$!prop.hint" != '')<p><span class="xHint">$prop.hint</span></p>#end$doc.display($prop.name, "edit", $obj)</dd>
```

Comment:
Should we escape the HTML from the hint?

Reply:
+1

Reply:
Indeed, fixed in https://github.com/xwiki/xwiki-platform/commit/dc6655f9222d88afe3301c383dd2a8224e693cd5.